package.json

```diff
@@ -58,7 +58,7 @@
 		"qunit": "2.21.0",
 		"rollup": "4.22.4",
 		"selenium-webdriver": "4.21.0",
-		"sinon": "9.2.4",
```

Comment:
I almost said something in the previous PR, but forgot the main branch still supports IE9.